fe/fe-core/src/main/java/com/starrocks/analysis/ColumnDef.java

```diff
@@ -86,7 +86,11 @@ public DefaultValueDef(boolean isSet, Expr expr) {
     private final TypeDef typeDef;
     private AggregateType aggregateType;
     private boolean isKey;
-    private boolean isAllowNull;
+    // For primary key column null constraint, when null specify, should report an error, when not null specify, should working as normal, when implicit specified, default behavior should be working as not null specify.
```

Comment:
Primary-key column should obey the not-null constraint. When creating a table, the not-null constraint will add to the primary-key column default. If the user specifies NULL explicitly, semantics analysis will report an error.